packages/shared/src/components/fields/TextField.tsx

```diff
@@ -137,6 +137,7 @@ export function TextField({
           },
           styles.field,
         )}
+        style={{ ...style, borderRadius: 'rounded-10' }}
```

Comment:
This think this is not right?
Not sure this is even needed here

Reply:
Weird, I removed it yesterday.